sidebars.js

```diff
@@ -568,33 +578,6 @@ const sidebars = {
         },
       ],
     },
-    {
```

Comment:
Removes the "Other Features" section from left sidebar of SQL reference (Introduction will get "Operators")